gradle.properties

```diff
@@ -9,5 +9,4 @@ kotlin.code.style=official
 #Android
 android.useAndroidX=true
 
-systemProp.org.gradle.internal.publish.checksums.insecure=true
-moko.resources.disableStaticFrameworkWarning=true
\ No newline at end of file
+systemProp.org.gradle.internal.publish.checksums.insecure=true
```

Comment:
FWIW, we can remove that now. It was an issue with bintray or sonatype but it's been fixed now. See https://github.com/gradle/gradle/issues/11308

Reply:
Yup, all good. See https://github.com/gradle/gradle/issues/11412#issuecomment-733655244

Reply:
I removed it in https://github.com/paug/openfeedback-sdk-kotlin/pull/29/commits/e22400f42b9d47fd7f166a920bd222e8c921505f

Reply:
A comment not easy to find! :D

Reply:
Yeaaaaaaaa, it's a secret handshake between lib developers :P